### ISSUE TITLE
没有添加F.tanh=Rp(F.tanh,_tanh)？但是添加进去也没有转换？

### ISSUE THREAD

**Issue by DJinsis** [xxradon/PytorchToCaffe#40]
你好，作者，我看你的代码没有添加F.tanh=Rp(F.tanh,_tanh)，但是我自己添加进去，还是不能实现转换，是什么问题呢？

**Comment by DJinsis** [comment]
@xxradon 是我添加不对吗？还是有其他注册操作
![image](https://user-images.githubusercontent.com/9247758/64174133-3d8df400-ce8b-11e9-9f3a-ccca1e765d35.png)

**Comment by xxradon** [comment]
我已经更新了tanh的转换，你可以试试。

**Comment by DJinsis** [comment]
早上我修改了一下，我建了一个简单的网络，如下，包含Conv2d，relu，以及Tanh
![image](https://user-images.githubusercontent.com/9247758/64222127-c21b5980-cf01-11e9-83ce-cc2838132344.png)
但是我跑了转换模型。转换后的结果是：
![image](https://user-images.githubusercontent.com/9247758/64222259-09094f00-cf02-11e9-887d-8954c58f23eb.png)
然后我删了tanh，尝试是不是因为网络最后一层不能加载，发现删了还是能的：
![image](https://user-images.githubusercontent.com/9247758/64222359-5e456080-cf02-11e9-9a4f-c3af2ef130c7.png)

**Comment by VIRGoMerz** [comment]
@DJinsis 请问大佬后来找到解决方案了吗，我发现sigmoid也有同样的问题

**Comment by ShockeenLee** [comment]
> @DJinsis 请问大佬后来找到解决方案了吗，我发现sigmoid也有同样的问题

是torch.sigmoid()，不是nn.sigmoid()

**Comment by luameows** [comment]
Trying to use nn.functional.sigmoid instead of nn.Sigmoid can solve this problem.
